src/emulate.c

```diff
@@ -1219,6 +1220,60 @@ RVOP(cswsp, {
 })
 #endif
 
+/* auipc + addi */
```

Comment:
Is it possible to manipulate the sequence `lui` + `addi`?
See https://github.com/sysprog21/rv32emu/issues/81

Disassembly of CoreMark:
```
   10324:       000087b7                lui     a5,0x8
   10328:       b0578793                addi    a5,a5,-1275 # 0x7b05
```

Reply:
It is possible. however, there are some problems when running qrcode.elf if we import this pattern, so I skip it in this pull request.

Reply:
Add a comment starting with "FIXME"

Reply:
> It is possible. however, there are some problems when running qrcode.elf if we import this pattern, so I skip it in this pull request.

Add a comment starting with "FIXME: lui + addi"